LondonDataServices.IDecide.Core/Migrations/20250813142558_AddInitialMigration.Designer.cs

```diff
@@ -12,7 +12,7 @@
 namespace LondonDataServices.IDecide.Core.Migrations
 {
     [DbContext(typeof(StorageBroker))]
-    [Migration("20250811132508_AddInitialMigration")]
+    [Migration("20250813142558_AddInitialMigration")]
     partial class AddInitialMigration
```

Comment:
This does not look correct.  There is already a migration "AddInitialMigration".  Can you remove this one and add one that decribes what you did?

i.e. add-migration UpdatedDecisionWithResponsiblePersonProps